flight_computer/platformio.ini

```diff
@@ -32,6 +32,7 @@ check_src_filters =
 build_unflags=-fno-rtti
 
 build_flags = 
+    '-D FIRMWARE_VERSION="3.0.0-RC1"'
```

Comment:
name this constant uniquely between the three projects

FC_FIRMWARE_VERSION
TELEMETRY_FIRMWARE_VERSION
GS_FIRMWARE_VERSION

or similar